package.json

```diff
@@ -63,7 +61,7 @@
     "d3-shape": "1.0.3",
     "d3-zoom": "1.0.2",
     "invariant": "2.2.4",
-    "lodash": "^4.17.15",
+    "lodash": "^4.17.19",
```

Comment:
I don't think it change something for real.
We can update it to specify some contract but it will not change the resolution in the host app anyway

Reply:
nvm, it's a dep 😂

Reply:
I think `lodash` could be a peer. Here we assure that even the host app will retrieve the last version, at least for this lib. But yeah the host app has to manage his own `lodash` version if different.

Reply:
> nvm, it's a dep 😂

Yep 👍 , the change preview was misleading 😉